### PR TITLE
Add WLP-Platform header to features

### LINKED ISSUE #2803: Productize sessionCache-1.0 and jcacheContainer-1.1
user story:
"As a user of Liberty, I would like the sessionCache-1.0 feature, currently in beta, to become fully supported GA feature so that I can use it in production to achieve HTTP session persistence."

Promote the sessionCache-1.0 feature to GA.
Promote the jcacheContainer-1.1 feature to GA.
Remove the include of bells-1.0 from jcacheContainer-1.1.
Add and review final documentation and examples.
Update any blog posts as needed.
If necessary, disable tracing of session data

### LINKED ISSUE #28203: Versionless Support for WLP-Platform feature header for assigning platform to feature
All public versioned features that belong to a "platform" like jakartaee-10.0 and microProfile-6.0 should add a feature header "WLP-Platform" that assigns an optional list of platforms versions.

This issue is split into two parts... 

1) internal api to read the header value - an example of this is here: https://github.com/OpenLiberty/open-liberty/pull/7997/files#diff-4d17e5616831354b0a1711b9be8908e73539153afd803b33c8102fe8231ce4da

The format should be straight forward list of platforms...
(example)
WLP-Platform: jakartaee-8.0,jakartaee-9.1

2) The actual adding of the values to the public features - Ask Ricky to generate a report on both MP and JakartaEE platforms to help finiding which versions for each feature.

### PR DESCRIPTION
**Pull request by jimblye**
Fixes #28203

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/noShip-1.0/io.openliberty.noShip-1.0.feature [hunk 1]
```diff
@@ -6,3 +6,4 @@ visibility=public
 kind=noship
 edition=full
 WLP-InstantOn-Enabled: true
+WLP-Platform: jakartaee-11.0
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/noShip-1.0/io.openliberty.noShip-1.0.feature]: should remove this one...

File: dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-9.1/io.openliberty.webProfile-9.1.feature [hunk 2]
```diff
@@ -30,3 +30,4 @@ Subsystem-Name: Jakarta EE Web Profile 9.1
 kind=ga
 edition=core
 WLP-InstantOn-Enabled: true
+WLP-Platform: jakartaee-9.1
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-9.1/io.openliberty.webProfile-9.1.feature:33]: remove this one?

File: dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-10.0/io.openliberty.webProfile-10.0.feature [hunk 3]
```diff
@@ -29,3 +29,4 @@ Subsystem-Name: Jakarta EE Web Profile 10.0
 kind=ga
 edition=core
 WLP-InstantOn-Enabled: true
+WLP-Platform: jakartaee-10.0
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-10.0/io.openliberty.webProfile-10.0.feature:32]: remove

File: dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-11.0/io.openliberty.webProfile-11.0.feature [hunk 4]
```diff
@@ -29,3 +29,4 @@ Subsystem-Name: Jakarta EE Web Profile 11.0
 kind=noship
 edition=full
 WLP-InstantOn-Enabled: true
+WLP-Platform: jakartaee-11.0
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/webProfile-11.0/io.openliberty.webProfile-11.0.feature:32]: remove

File: dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-2.0/com.ibm.websphere.appserver.appSecurity-2.0.feature [hunk 5]
```diff
@@ -16,3 +16,4 @@ Subsystem-Name: Application Security 2.0
 kind=ga
 edition=core
 WLP-InstantOn-Enabled: true
+WLP-Platform: javaee-7.0,javaee-8.0
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-2.0/com.ibm.websphere.appserver.appSecurity-2.0.feature]: can you double check these values?  I see appSecurity-3.0 also uses javaee-8.0

File: dev/com.ibm.websphere.appserver.features/visibility/public/noShip-1.0/io.openliberty.noShip-1.0.feature (hunk 1 above)
Comment (jimblye) [reply dev/com.ibm.websphere.appserver.features/visibility/public/noShip-1.0/io.openliberty.noShip-1.0.feature]: done

File: dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-2.0/com.ibm.websphere.appserver.appSecurity-2.0.feature (hunk 5 above)
Comment (jimblye) [reply dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-2.0/com.ibm.websphere.appserver.appSecurity-2.0.feature]: Removed javaee-8.0

**Comment by cbridgha** [comment]
#build

**Review by cbridgha — DISMISSED** [review]
looks good

**Review by hlhoots — DISMISSED** [review]
LGTM

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/com.ibm.websphere.appserver.restConnector-2.0.feature [hunk 6]
```diff
@@ -27,3 +27,4 @@ Subsystem-Name: Admin REST Connector 2.0
 kind=ga
 edition=core
 WLP-InstantOn-Enabled: true
+WLP-Platform: jakartaee-9.1
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/com.ibm.websphere.appserver.restConnector-2.0.feature]: remove restConnector feature

**Comment by jimblye** [comment]
#build

**Review by cbridgha — APPROVED** [review]
Thanks looks good

**Review by cbridgha — APPROVED** [review]
looks good

**Comment by cbridgha** [comment]
#libby

**Pull request merged**